expan/core/statistics.py

```diff
@@ -9,6 +9,13 @@ def _delta_mean(x, y):
 	"Implemented as function to allow calling from bootstrap."
 	return np.nanmean(x) - np.nanmean(y)
 
+def make_delta(assume_normal=True, percentiles=[2.5, 97.5],
```

Comment:
If `delta()` is a function, then `make_delta()` creates a `delta()` function on the fly and returns it -- seems logical to me :~)